CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # 2020-01-08
 
+## API changes (relevant for self-hosters and client developers)
+
+- Allow up to 256 characters as handle, dots and dashes too (#953)
+- Refuse to delete non-empty IdPs (412 precondition failed) (#875)
+- Make sure that someone is SSO user before setting ManagedBy (#947)
+- Linear onboarding feature: Provide information about custom backends (#946)
```

Comment:
Listing the new API endpoints in the changelog would be nice.

Reply:
Good idea 👍